src/data-access/facades/repository-facade.ts

```diff
@@ -69,25 +69,27 @@ import { ShowStyle } from '../../model/entities/show-style'
 import {
   MongoShowStyleConfigurationChangedListener
 } from '../repositories/mongo/mongo-show-style-configuration-changed-listener'
+import { Database } from '../repositories/interfaces/database'
 
 export class RepositoryFacade {
+
+  public static getDatabase(): Database {
+    return MongoDatabase.getInstance(LoggerFacade.createLogger())
+  }
+
   public static createRundownRepository(): RundownRepository {
     const mongoRundownRepository: RundownRepository = new MongoRundownRepository(
-      RepositoryFacade.getMongoDatabaseInstance(),
+      MongoDatabase.getInstance(LoggerFacade.createLogger()),
```

Comment:
Would it make sense to extract this into a helper method, since it is used in 30 places?

Reply:
Yes and no.
It would be a slippery slope to create tons of helper methods for very small things. And in this case we are "simply" calling our "facade setup".

I did toy with the idea, in times past, to make the logger parameter optional on `getInstance` and make the method throw if the logger isn't provided on the very first call of `getInstance`, so subsequent calls wouldn't need to provide it.
But not sure if that's a good idea.